boards/arm/mps2_an521/mps2_an521.dts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2018-2019 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <arm/armv8-m.dtsi>
+#include <dt-bindings/i2c/i2c.h>
+
+/ {
+	compatible = "arm,mps2";
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m33";
+			reg = <0>;
```

Comment:
If #12173 is accepted, we might want to add here the mpu configuration (if mpu exists, of course)